src/python/WMCore/REST/Format.py

```diff
@@ -244,6 +244,7 @@ def stream_chunked(self, stream, etag, preamble, trailer):
                 trailer = None
                 raise
             finally:
+                print("ERROR, json.dumps failed to serialize %s, type %s" % (obj, type(obj)))
```

Comment:
Alan, there is no general except: block here, the code only contains except GeneratorExit: which will be triggered ONLY when generator will exhaust. And, I don't know if except is desired here. But finally: will certainly called if exception was thrown. That's why I thought it is appropriate place.

Regarding cherrypy.log I think it is overhead anyhow, prints work more reliably and allows to get what you want without web backend library.